### PR TITLE
[Backend] API Testing for admin login #81 #251

### PR DESCRIPTION
**Pull request by kenkirito**
## Issue that this pull request solves

 Closes: # (issue number)

## Types of changes

_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update (Documentation content changed)
- [ ] Other (please describe): 

## Checklist

_Put an `x` in the boxes that apply_

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] My changes does not break the current system and it passes all the current test cases.

## Screenshots

Please attach the screenshots of the changes made in case of change in user interface

## Other information

Any other information that is important to this pull request

### CODE REVIEW

**Comment by kenkirito** [comment]
[Backend] API Testing for admin login #81

**Comment by jackfrost13** [comment]
The tests are still failing 
`6/7` and `0/1` passed

**Comment by kenkirito** [comment]
test are running fine by my side

File: postman/Login_test [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+									"});\r",
+									"\r",
+									"/*signin*/\r",
+									"pm.test(\"Response has a data property\", function () {\r",
+									"    var jsonData = pm.response.json();\r",
+									"    pm.expect(jsonData).to.be.a('object');\r",
+									"});\r",
+									"\r",
+									"\r",
+									"pm.test(\"400 invalid request\", function () {\r",
```
Comment (jackfrost13) [inline postman/Login_test:60]: this is failing

Comment (jackfrost13) [reply postman/Login_test:60]: I see you have provided a different image, those tests are not provided in the PR!
Moreover, the coverage is not what we were looking forward to

**Comment by kenkirito** [comment]
please check now @jackfrost13

**Review by jackfrost13 — COMMENTED** [review]
Firstly, it is not possible for us if you open a new PR every single time. You are supposed to update the files according to the changes we request. We cannot review 2-3 PR's for one issue. Moreover, every most of your PR's do not contain the requested changes so it is really not possible for us to continue like this.
The second request doesn't make any sense at all where you have just added one check for status code to be 200. 
Regex check is not inside any test.
I asked to check for JWT payload which is also absent.
Coverage is low after repeatedly telling you that you need to add more request for different scenario's

**Comment by jackfrost13** [comment]
Closing this because we cannot spend more time on this, I need to incorporate all the API testing which has been on halt due to this. Apologies if this didn't go as expected for you.
try taking up other issues if you are interested

**Pull request closed without merging**